src/sidebar/config/test/fetch-config-test.js

```diff
@@ -47,8 +47,8 @@ describe('sidebar/config/fetch-config', () => {
       // by h with the settings from `window.hypothesisConfig` in the parent
       // window.
       it('adds the apiUrl to the merged result', async () => {
-        const mergedConfig = await fetchConfig({});
-        assert.deepEqual(mergedConfig, { apiUrl: fakeApiUrl() });
+        const SidebarSettings = await fetchConfig({});
```

Comment:
```suggestion
        const sidebarSettings = await fetchConfig({});
```

This looks like a find and replace foobar. Same for the change below.